src/Form/TranslationRequestForm.php

```diff
@@ -46,7 +46,10 @@ public function save(array $form, FormStateInterface $form_state) {
    * @see TranslationFormTrait::translationFormElement()
    */
   public static function validateMaxLength(array $element, FormStateInterface &$form_state): void {
-    if (isset($element['#max_length']) && ($element['#max_length'] < mb_strlen($element['#value']))) {
+    // The value can be nested in a rare case in which the element is a
+    // text_format.
+    $value = $element['#value'] ?? $element['value']['#value'];
```

Comment:
In case we have a text format element with max lenght, the value is nested, so we need to account for this. No test added now because discovered another (bigger issue) which we might fix after, so the text would come then.